README.md

```diff
@@ -127,7 +127,7 @@ The compilation will create the `public/js/main.js` we configured above (`:main`
 
 We also need a simple HTTP server to serve our HTML since modern Browsers all place a few restrictions on files loaded directly from disk which will lead to issues later. `shadow-cljs` provides such a server but you can use anything you like at this point. It only matters that the files from the `public` directory are served properly. To start the built-in web server just adjust the build config from above.
 
-```
+```clojure
```

Comment:
`edn`?

Reply:
it's a clojure code, not?

Reply:
EDN is data notation inside Clojure.

Reply:
hum...https://github.com/edn-format/edn?

Reply:
This is just for syntax highlighting so it doesn't really matter. Also not sure if Github even highlights edn.